gdal/port/cpl_vsil_hdfs.cpp

```diff
@@ -61,6 +61,16 @@ CPL_CVSID("$Id$")
 /* ==================================================================== */
 /************************************************************************/
 
+#define SILENCE(expr) {\
+    int hOldStderr = dup(2);\
```

Comment:
in case the process run short of file handles available, dup() and open() could fail. It would be prudent to check them

Reply:
Okay, done.